### ISSUE TITLE
Button for automatic regrading should not be hidden behind a modal dialog

### ISSUE THREAD

**Issue by Mankro** [apluslms/a-plus#895]
In the new inspect submission page (A+ v1.11), sending a submission to regrading requires two clicks instead of one. One click on the regrade button and another inside the modal dialog. The modal dialog hinders the user experience. Regrading should be possible with one click. Since the new UI layout has little space, the long help/description text could be kept in a modal dialog or a tool tip maybe. For example, there could be two buttons, one for regrading (only one click required) and another button for reading the regrading instructions.

Teacher's request (internal ticket): https://rt.cs.aalto.fi/Ticket/Display.html?id=19964
